include/fmt/format.cc

```diff
@@ -280,9 +280,16 @@ const uint64_t internal::basic_data<T>::POWERS_OF_10_64[] = {
 
 FMT_FUNC internal::utf8_to_utf16::utf8_to_utf16(string_view s) {
   static const char ERROR_MSG[] = "cannot convert string from UTF-8 to UTF-16";
-  if (s.size() > INT_MAX)
+  if (s.size() > INT_MAX || s.data() == nullptr)
```

Comment:
Why add a nullptr check? Won't `MultiByteToWideChar` return an error in this case already?

Reply:
True, but without this check the function will now succeed for invalid input of `nullptr` and `0` length (because the size check precedes the call to `MultiByteToWideChar`). Clearly this is not the best behavior. Moreover, it fails some tests (UTF16ToUTF8Error/UTF8ToUTF16Error/UTF16ToUTF8Convert).

Reply:
Why is `nullptr` and `0` invalid? If size is zero than the data can be anything including nullptr.

Reply:
Well, this is a conceptual discussion, and probably a matter of definition and taste... Personally, I think that `nullptr` is a special case reserved for "invalid data" and this fact should not be silently discarded. I don't see significant advantages to the other approach, but if you prefer it, the code can be easily changed by reverting this check.
However, please note that the above mentioned tests assume that (nullptr, 0) is invalid input. So, if we decide that it becomes valid, we need to redefine the tests logic.

Reply:
Empty string view has the null data pointer (https://godbolt.org/g/JtWcst), so I suggest removing the `nullptr` check here.

Reply:
Ok, good point :-)
Thanks!